etc/ReplayOfflineConfiguration.py

```diff
@@ -657,6 +684,23 @@
                physics_skims=["ZMu", "EXODisappTrk", "LogError", "LogErrorMonitor", "EXOCSCCluster", "EXODisappMuon"],
                scenario=ppScenario)
 
+DATASETS = ["PPRefDoubleMuon0", "PPRefDoubleMuon1", "PPRefDoubleMuon2", "PPRefDoubleMuon3"]
+
+for dataset in DATASETS:
+    addDataset(tier0Config, dataset,
+               do_reco=True,
+               write_reco=False,
+               write_dqm=True,
+               tape_node="T1_US_FNAL_MSS",
+               disk_node="T1_US_FNAL_Disk",
+               alca_producers=["TkAlMuonIsolated", "HcalCalIterativePhiSym", "MuAlCalIsolatedMu",
+                               "HcalCalHO", "HcalCalHBHEMuonProducerFilter",
+                               "SiPixelCalSingleMuonLoose", "SiPixelCalSingleMuonTight",
+                               "TkAlZMuMu", "TkAlDiMuonAndVertex", "TkAlJpsiMuMu", "TkAlUpsilonMuMu"],
```

Comment:
I suggest to only keep the double muon producers for the DoubleMuon ppRef PDs
```suggestion
               alca_producers=["TkAlZMuMu", "TkAlDiMuonAndVertex", "TkAlJpsiMuMu", "TkAlUpsilonMuMu"],
```

(side note: personally I think it's quite inconvenient to keep swinging from merged Muon to split SingleMuon+DoubleMuon PDs...)

Reply:
I guess it is a question for @saumyaphor4252. Which ones should we keep for DoubleMuon PPRef. (I agree about the splitting of Muon PDs)

Reply:
Ok, I confirm we will get riid of hte other producers.